### PR TITLE
Fix docs generation for extra labels and annotations

### PR DESCRIPTION
**Pull request by ashleywang1**
BOT NOTES: 
resolves https://github.com/solo-io/skv2/issues/246

### CODE REVIEW

File: codegen/model/chart.go [hunk 1]
```diff
@@ -124,12 +124,12 @@ type Values struct {
...
 	Env          []v1.EnvVar              `json:"env" desc:"Specify environment variables for the deployment. See the [Kubernetes documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#envvarsource-v1-core) for specification details." omitChildren:"true"`
 
-	CustomPodLabels             map[string]string `json:"customPodLabels,omitempty" desc:"Custom labels for the pod"`
-	CustomPodAnnotations        map[string]string `json:"customPodAnnotations,omitempty" desc:"Custom annotations for the pod"`
-	CustomDeploymentLabels      map[string]string `json:"customDeploymentLabels,omitempty" desc:"Custom labels for the deployment"`
-	CustomDeploymentAnnotations map[string]string `json:"customDeploymentAnnotations,omitempty" desc:"Custom annotations for the deployment"`
-	CustomServiceLabels         map[string]string `json:"customServiceLabels,omitempty" desc:"Custom labels for the service"`
-	CustomServiceAnnotations    map[string]string `json:"customServiceAnnotations,omitempty" desc:"Custom annotations for the service"`
+	CustomPodLabels             map[string]string `json:"extraPodLabels,omitempty" desc:"Custom labels for the pod"`
```
Comment (EItanya) [inline codegen/model/chart.go:127]: Can we change the struct names so they match?

**Review by EItanya — CHANGES_REQUESTED** [review]
Confusing mismatch of `json` name and struct name

File: codegen/model/chart.go (hunk 1 above)
Comment (ashleywang1) [reply codegen/model/chart.go:137]: Sure! The intention was to differentiate between the labels set by the user (using some helm values.go file) and labels set by the operator (programmatically) - so there'll be another confusing assignment, but it'll be in a different area.

**Review by ilackarms — APPROVED** [review]
👍  🎉  thanks for this fix!

**Pull request merged**